source/_open_source.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="open_source container main-colors" id="open_source">
+<h4 class="section-title">Open Source Software</h4>
+  <div class="row">
+  	<div class="col s12 m6">
+      <div class="icon-block">
+        <h2 class="center"><i class="material-icons md-48"><img src="images/chair-school.png"></i></h2>
+        <h4 class="center">
+          <a href="https://github.com/SavvySoftWorksLLC/body-snatchers">Brown Noser</a>
```

Comment:
We will need to link to the `Brown Noser` repo, I thought we had one in SSW, we will have to ask Paul